src/org/whstsa/library/LibraryDB.java

```diff
@@ -43,10 +43,10 @@ public void start(Stage stage) {
             }
         }
         this.loadJSON((arg0) -> {
-            LOGGER.debug("JSON Loaded.");
```

Comment:
When you're done using the command debugging suite make sure that you re-enable the GUI before committing or making a PR to master